templates/taxi/car_detail.html

```diff
@@ -18,5 +18,16 @@
     {% for driver in car.drivers.all %}
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
+  {% if user in car.drivers.all %}
+  <form method="post" action="{% url 'car-remove-driver' car.pk user.pk %}">
+    {% csrf_token %}
+    <button type="submit" class="btn btn-danger">Delete me from this car</button>
+  </form>
+{% else %}
+  <form method="post" action="{% url 'car-assign-driver' car.pk user.pk %}">
```

Comment:
The URL pattern 'car-assign-driver' is used here, but ensure that this URL pattern is correctly defined in your `urls.py` file. Also, verify that the view handling this URL expects both `car.pk` and `user.pk` as parameters.